packages/sources/iex-cloud-test/src/stock-utils.ts

```diff
@@ -0,0 +1,124 @@
+import { customSettings } from './config'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'asset', 'symbol'],
+    description: 'The symbol to query',
+    required: true,
+    type: 'string',
+  },
+} as const
```

Comment:
After the typescript upgrade and the fw bump past 0.25, we want to move to using the satisfies operator here like
```suggestion
} satisfies InputParameters
```